### PR TITLE
fix: add loading to videoblockeditorposterlibrary

### PR DESCRIPTION
**Pull request by tanflem**
# Description

### <samp>🤖 Generated by Copilot at 6f27118</samp>

Added loading indicator for poster image selection in video block editor. This improves the user feedback and experience when creating or updating a video block with a custom poster image.

- [Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/33281894/todos/6385169132)

# How should this PR be QA Tested?

Checked that loading icon appeared when uploading a background image 

- [ ] Test A
- [ ] Test B

# Walkthrough

### <samp>🤖 Generated by Copilot at 6f27118</samp>

*  Add `loading` prop to `ImageLibrary` component to show loading indicator while poster image is being created or updated ([link](https://github.com/JesusFilm/core/pull/1739/files?diff=unified&w=0#diff-67bb64174c7324e212c1ea7033becff02802c6be9d9f37d42695a6f571790a0dR233))

### CODE REVIEW

File: apps/journeys-admin/src/components/Editor/VideoBlockEditor/Settings/Poster/Library/VideoBlockEditorSettingsPosterLibrary.tsx [hunk 1]
```diff
@@ -230,6 +230,7 @@ export function VideoBlockEditorSettingsPosterLibrary({
       onClose={onClose}
       onChange={handleChange}
       onDelete={deleteCoverBlock}
+      loading={createLoading || updateLoading}
```
Comment (GeronimoJohn) [inline apps/journeys-admin/src/components/Editor/VideoBlockEditor/Settings/Poster/Library/VideoBlockEditorSettingsPosterLibrary.tsx:233]: would you be able to add a test for it?

Comment (tanflem) [reply apps/journeys-admin/src/components/Editor/VideoBlockEditor/Settings/Poster/Library/VideoBlockEditorSettingsPosterLibrary.tsx:233]: For imageLibrary or VideoBlockEditorSettingsPosterLibrary?

Comment (tanflem) [reply apps/journeys-admin/src/components/Editor/VideoBlockEditor/Settings/Poster/Library/VideoBlockEditorSettingsPosterLibrary.tsx:233]: All the functional code was already there. I just passed the prop to ImageLibrary. It looks like ImageLibrary does not have a test for loading though. I could add one there.

Comment (GeronimoJohn) [reply apps/journeys-admin/src/components/Editor/VideoBlockEditor/Settings/Poster/Library/VideoBlockEditorSettingsPosterLibrary.tsx:233]: For VideoBlockEditorSettingsPosterLibrary, and yea I think it is missing a loading test. If you could add one please

**Review by GeronimoJohn — APPROVED** [review]

**Pull request merged**